### PR TITLE
chore: allow pre-releases

### PR DESCRIPTION
**Pull request by ehmicky**
This allows using pre-releases like our other repositories.

### CODE REVIEW

File: .github/workflows/pre-release.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          cache: 'npm'
+          check-latest: true
+          cache-dependency-path: 'npm-shrinkwrap.json'
+          registry-url: 'https://registry.npmjs.org'
+      # This can be removed once npm >=7.20.0 is installed by default.
+      # See https://github.com/npm/cli/issues/2755.
+      - name: Upgrade npm
+        run: npm install -g npm@latest
+      - name: Install core dependencies
+        run: npm ci --no-audit
```
Comment (erezrokah) [inline .github/workflows/pre-release.yml:24]:
```suggestion
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: '*'
          cache: 'npm'
          check-latest: true
          registry-url: 'https://registry.npmjs.org'
```

We should match the other steps in this repo workflows

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks @ehmicky, I added a comment.

We should align the pre-release with the other workflows in this repo

File: .github/workflows/pre-release.yml (hunk 1 above)
Comment (ehmicky) [reply .github/workflows/pre-release.yml]: Oops! Is this ok now after eedae1c09555c17d6ad883c87d1641afb91e1af8?

**Review by erezrokah — APPROVED** [review]
🚀

**Pull request merged**